saylua/modules/messages/messages.py

```diff
@@ -24,29 +45,19 @@ def messages_main_post():
     user_message_ids = request.form.getlist('user_conversation_id')
     keys = []
     for m_id in user_message_ids:
-        m_id = make_ndb_key(m_id)
         if not m_id:
             flash('You are attempting to edit an invalid message!', 'error')
             return redirect('/messages/', code=302)
         keys.append(m_id)
 
-    user_messages = ndb.get_multi(keys)
-    for m in user_messages:
-        if not m:
-            flash('You are attempting to edit a message which does not exist!', 'error')
-            return redirect('/messages/', code=302)
-        if m.user_id != g.user.id:
-            flash('You do not have permission to edit these messages!', 'error')
-            return redirect('/messages/', code=302)
+    if ('delete' in request.form):
+        result = delete_conversations(keys)
```

Comment:
If one user deletes a message, it's deleted for all participating users as well?